scripts/version.sh

```
@@ -0,0 +1,153 @@
+get_current_version() {
+  if [ -f "$VERSION_FILE" ]; then
+    cat "$VERSION_FILE"
+  else
+    echo "$DEFAULT_VERSION"
+  fi
+}
+
+# Function to create a new version
+generate_version() {
+  local version_type=$1
+  local current=$(get_current_version)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)